xonsh/__init__.py

```diff
@@ -1,5 +1,11 @@
 __version__ = '0.5.3'
 
+import sys as _sys
+if '--timings' in _sys.argv:
+    from xonsh.timings import setup_timings
+    setup_timings()
+del _sys
```

Comment:
I think this should be removed because it breaks amalgamation (`xonsh.timings` is imported and then overwritten later). The `setup_timings()` call can then be moved to main.py, right before the first event is fired.  

Also, given the way that out clock works with `resource.getrusage()`, it is actually better to assume that the first time is 0.0. That will capture the Python interpreter startup time. This version doesn't.